templates/metaboxes/fields/reorderable-multiselect.blade.php

```diff
@@ -0,0 +1,28 @@
+<div class="form-field">
+	<pressbooks-reorderable-multiselect data-messages="{{ json_encode([
+		'Add' => __('Add', 'pressbooks'),
+		'Remove' => __('Remove', 'pressbooks'),
+		'Move Up' => __('Move Up', 'pressbooks'),
+		'Move Down' => __('Move Down', 'pressbooks'),
+		'Available Options' => __('Available options', 'pressbooks'),
+		'Moved to position $1' => __('Déplacé en position $1'),
+		'Removed $1 from selection' => __('Supprimer $1 de la sélection'),
+		'Added $1 to selection' => __('Ajouté $1 à la sélection')
```

Comment:
Is there any reason why those are defaulted to French? 🤔

Reply:
Oops, I was testing those strings. They should be English. I'll fix that.